app/code/Codifi/Sales/ViewModel/Order/View/Info.php

```diff
@@ -42,19 +41,16 @@ public function __construct(
      *
      * @param Order $currentOrder
      * @return string
-     * @throws LocalizedException
      */
     public function getAttributeLabel(Order $currentOrder): string
     {
-        $attributeLabel = '';
         $orderType = $currentOrder->getData(Config::ORDER_TYPE_CODE);
-        $options = $this->orderTypeSource->getAllOptionsOrderType();
-        foreach ($options as $item) {
-            if ($item['value'] === $orderType) {
-                $attributeLabel = $item['description'] ?? $item['value'];
-            }
+        if ($orderType) {
```

Comment:
See same logic implementation in app/code/Codifi/Sales/Block/Adminhtml/Order/Renderer/Type::render
Code can be simplified